docs/components/modeler/web-modeler/launch-cloud-modeler.md

```diff
@@ -0,0 +1,29 @@
+---
+id: launch-cloud-modeler
+title: Launch Web Modeler
+---
+
+Since Web Modeler is embedded in the Camunda Cloud Console, it does not need to be started separately.
+
+To launch Web Modeler, follow the steps below:
+
+1. Click on the **Modeler** tab at the top of the page
+![cloud web modeler menu item](img/cloud-web-modeler-menu-item.png)
+
+2. Click **New Project** to create a new project to store all your diagrams
+![web modeler empty home](img/web-modeler-new-user-new-project.png)
+:::note
+You can name your project now or simply continue your journey
+:::
+
+3. Click **New** and choose **BPMN Diagram**
+![web modeler empty project](img/web-modeler-new-user-new-diagram-choose-bpmn.png)
+:::note
+You can name your diagram now or simply continue your journey
+:::
+
+4. Choose an existing template (i.e. Absence Request) or **Create blank**
+![web modeler new diagram modal](img/web-modeler-new-user-new-diagram-modal.png)
+
+5. Congrats! You just created your first Diagram
```

Comment:
Same as https://github.com/camunda-cloud/camunda-cloud-documentation/pull/620/files#r813829741, "Diagram" is capitalized while two bullet points above it isn't.